dev/builder/build.sh

```diff
@@ -57,7 +57,7 @@ echo "Starting CKBuilder..."
 
 JAVA_ARGS=${ARGS// -t / } # Remove -t from args.
 
-VERSION="4.14.1 TAO-1"
+VERSION="4.15.1 TAO-1"
```

Comment:
**issue:** The version of CKE must remain in sync with the base repo.

**suggestion:** I'd rather suggest to increase the TAO version
```suggestion
VERSION="4.14.1 TAO-2"
```